mac-desktop.yml

```diff
@@ -0,0 +1,36 @@
+---
+- hosts: all
+  connection: local
+
+  vars_files:
+    - default.config.yml
+
+  # base: uninstalls, command-line, homebrew
+  # rails: rbenv, gems, postgresql
+  # php: phpunit, wp-cli
+  # apps: github, signage, blogs, inside
+  roles:
```

Comment:
@tatwell I forgot to mention I added those group tags to the roles but also left the individual ones so they can either be run individually or in a group in case for whatever reason only one needs to be run it will run faster